### PR TITLE
feat: allow to disable markdown shortcuts

### PR DESCRIPTION
**Pull request by d0zingcat**
Related: #4022

### CODE REVIEW

**Comment by d0zingcat** [comment]
Except for issue has discussed about, I have to give a brief illustration about this PR.
Although the best way(to keep code clean and easy to understand) is that we should use `"EnableKeymap"` like option which defaults to `false`, if so most users who are used to the keymaps and have upgraded their memos will have to enable this option manually(even though we can issue one new notice to notice this update).
So I have to use another way, whereas we can give the option to disable this keymap.
BTW, I've added some locales I've been known of but the others I don't have any ideas. Should I use translator to fill them or wait for another PR to update?

File: proto/api/v1/workspace_setting_service.proto [hunk 1]
```diff
@@ -115,6 +115,8 @@ message WorkspaceMemoRelatedSetting {
   string default_visibility = 9;
   // reactions is the list of reactions.
   repeated string reactions = 10;
+  // disable_markdown_keymap disallow the registration of markdown keymap.
+  bool disable_markdown_keymap = 11;
```
Comment (boojack) [inline proto/api/v1/workspace_setting_service.proto:119]: How about `disable_keyboard_shortcuts`?

image

https://code.visualstudio.com/docs/getstarted/keybindings#_keyboard-shortcuts-editor

**Review by boojack — APPROVED** [review]
LGTM, otherwise

File: proto/api/v1/workspace_setting_service.proto (hunk 1 above)
Comment (d0zingcat) [reply proto/api/v1/workspace_setting_service.proto:119]: Make sense. How about `disable_markdown_keyboard_shortcuts` as it only suppress the markdown related shortcuts(Yeah it does not affect the command+enter to submit)?

Comment (boojack) [reply proto/api/v1/workspace_setting_service.proto:119]: Then it's recommand to use `disable_markdown_shortcuts` which is simpler and easier to understand.

**Review by boojack — APPROVED** [review]
LGTM, thanks for the contribution

**Pull request merged**